### ISSUE TITLE
[JS][Accessibility] [ac-horizontal-separator does not set "aria-hidden" to true]

### ISSUE THREAD

**Issue by niravk-microsoft** [microsoft/AdaptiveCards#5073]
# Platform

- [ ] JavaScript

# Author or host

We are using JSRenderer to render Adaptive cards

# Version of SDK

Adaptive cards 2.5.0

# Details

JSRenderer is adding "ac-horizontal-separator" however it does not have "aria-hidden" property set as true which results into readers are reading this empty div which is not required to be read by reader.

**Comment by dclaux** [comment]
@niravk-microsoft I am not able to repro this. "aria-hidden" is already correctly set to true in the 2.5 release (and that was already true in 2.4 IIRC):
![image](https://user-images.githubusercontent.com/1334689/99116282-9cc7b800-25a8-11eb-9ed8-7d5f333221a3.png)

**Comment by niravk-microsoft** [comment]
Yes true. Actually we were using 2.4.0 version. with 2.5.0 its working and hence closing this now.
